Remove loading screen

domenicw1 opened the pull request:
There is a massive amount of users using Win95.css, and they all have a loading screen whenever you visit their profile. However, for anyone wanting to visit the profile or even just going through random, this is very annoying. As most users import this css directly, this would resolve this issue.

myspace93 commented:
What the hell is wrong with you. That is the only part of those themes that aren't written by me. Also public CSS on the internet cannot even be under copyright, a brand identity etc can. You have nothing to complain about. If you asked I would have removed it. Jeez. No reason to be anal about it.

I can probably however claim YOU if you use my code because I haven't released my rights to it.

I can merge if you delete your repo and stop calling me a thief. Thanks, that would be enjoyed.

I'm only doing this for fun and being creative because I enjoy the community, not  to be humiliated over 10 lines of CSS out of 700 lines in the damn repo. Like what the hell dude, way to kill the fun.

myspace93 commented:
I have removed it myself, if you would delete your repo that would be appreciated.

domenicw1 committed:
Remove loading screen

There is a massive amount of users using this css, and they all have a loading screen whenever you visit their profile. However, for anyone wanting to visit the profile or even just going through random, this is very annoying. As most users import this css directly, this would resolve this issue.

domenicw1 commented:
pinging @Tucanix

myspace93 commented:
Could also have shortened the duration, but since it annoyed you enough to make a pull request I'll merge it. Thanks.
